k9mail-library/src/main/java/com/fsck/k9/mail/Address.java

```diff
@@ -95,7 +95,8 @@ public String getPersonal() {
         return mPersonal;
     }
 
-    public void setPersonal(String personal) {
+    public void setPersonal(String newPersonal) {
+        String personal = newPersonal;
```

Comment:
What's the point of this?

Related note, this method is doing a check for empty string and *then* trims, so it can still result in an empty string being assigned.